pkg/interface/src/apps/groups/components/lib/contact-sidebar.tsx

```diff
@@ -166,8 +169,20 @@ export class ContactSidebar extends Component<ContactSidebarProps, ContactSideba
           >Channels</Link>
           {shareSheet}
           <h2 className="f9 pt4 pr4 pb2 pl4 gray2 c-default">Members</h2>
-          {contactItems}
-          {groupItems}
+          {this.memberbox.current ? <List
+            height={this.memberbox.current.getBoundingClientRect().height}
+            className="flex-auto"
+            itemCount={contactItems.length + groupItems.length}
+            itemSize={48}
```

Comment:
`itemSize={48}` is based on a padding of 6 around a sigil of 32 which is not 48 damn it